janus_core/calculations/md.py

```diff
@@ -1517,6 +1515,74 @@ def default_formats(self) -> dict[str, str]:
         return super().default_formats | {"Target_T": ".5f"}
 
 
+class NVT_Bussi(NVT):  # noqa: N801 (invalid-class-name)
```

Comment:
i know in ase they call it bussi but we shall call it CSVR that is the name in the paper iirc, and how is names in cp2k too.